usage-examples/ether-transactions/transfer-ether.js

```diff
@@ -0,0 +1,62 @@
+
+// Scroll to the bottom to see how the transfer example can be triggered
+
+// dotenv is an external dependency and would need to be installed for your project
+// e.g. npm i -s dotenv
+require('dotenv').config() // this ensures process.env. ... contains your .env file configuration values
+
+const Web3 = require('web3');
+
+const web3 = new Web3(new Web3.providers.HttpProvider(process.env.PROVIDER_URL));
+
+
+async function signAndSend(transactionObject, senderPrivateKey) {
+
+    const signedTransaction = await web3.eth.accounts.signTransaction(transactionObject, senderPrivateKey)
+
+    web3.eth.sendSignedTransaction(signedTransaction.rawTransaction)
+        .on('transactionHash', (hash) => {
+            console.log(hash);
+        })
+        .on('receipt', (receipt) => {
+            console.log(receipt);
+        })
+        .on('confirmation', (confirmationNumber, receipt) => {
+            console.log(confirmationNumber);
+            console.log(receipt);
+        })
+        .on('error', console.error)
+
+}
+
+
+async function getTransactionObject(from, to, value) {
+
+    const txCount = await web3.eth.getTransactionCount(from, "pending")
+    const gasPrice = await web3.eth.getGasPrice()
+    
+    return {
+        nonce: web3.utils.numberToHex(txCount),
+        gasLimit: web3.utils.numberToHex(33000),
+        gasPrice: web3.utils.toHex(gasPrice),
+        from,
+        to,
+        value,
+        chainId: '0x1',
+    }
+
+}
+
+
+async function triggerTransferExample() {
+
+    const amountInEtherToBeSent = 0.01 * 1000000000000000000 // this would e.g. transfer 0.01 Ether
```

Comment:
Can you make this a little bit easier to read? Maybe include a function wei() and call it instead? And maybe do in the function `Math.pow(10, 18);`      
I like if it's easier readable. It also prevents copy & paste bugs.

Reply:
@koraykoska: Would it be cooler to introduce a holistic unit converter - allowing to convert like https://www.etherchain.org/tools/unitConverter ?

If yes it would be cool if we could implement this in a different PR in order to keep things simple - step by step.

Reply:
Michael, for now if you wouldn't mind updating `1000000000000000000`, I agree with Koray that it makes it a bit easier to read :)